src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
Since resources come at a premium in the firmware, doing something that can be achieved just as easy by existing configuration facilities is a bit doubtful. And I honestly can't see people wanting to assign 2 channels and switches to dealing with 3D or flipping over crashed quads.

I can see the use case for '3D by switch' though (assuming what you do is flipping the quad while in 'normal' mode, pulling the switch to reverse thrust, then when done flipping back and resetting the switch to go 'normal' again).

This would probably better be done as a separate mode though, we could have 'normal', '3D' (what 3D mode is now), and 'switched 3D' (what you are proposing). This could be preselected in the config. In '3D', everything would be as it is. In 'switched 3D', `BOX3DDISABLESWITCH` would switch rotation between forward and reverse, with the throttle being zero to full thrust in the same way for both directions. What do you think?